src/process/ProcessManagerImpl.cpp

```diff
@@ -266,7 +247,7 @@ ProcessExitEvent::Impl::run()
     {
         if (si.hStdOutput != NULL)
         {
-            CloseHandle(si.hStdOutput);
+            C loseHandle(si.hStdOutput);
```

Comment:
oops :)

Reply:
wat, I actually built this on windows and it worked. my linux laptop must have added a space in revenge for using another OS. I'll fix this later today and I also want to add that "multi process manager" test.